pmtiles/intro.qmd

```diff
@@ -0,0 +1,97 @@
+---
+title: PMTiles
+subtitle: Guidelines for PMTiles
+---
+
+# PMTiles
+
+PMTiles is a single-file archive format for tiled data, usually used for visualization.
+
+PMTiles can be used for any format of tiled data. It's used most often with vector data, where each tile is encoded as a Mapbox Vector Tile (MVT), but can also be used with e.g. raster data or terrain mesh data.
+
+## Analytical vs tiled data formats
+
+To understand PMTiles, it's important to understand the difference between "analytical" data and "tiled" data. Analytical data refers to data in its original form, without any modifications to geometry. Tiled data formats apply a variety of modifications to geometries, including clipping and simplification, to save space and make it faster to visualize.
+
+![](../images/analytical_vs_tiled_geoms.png)
+
+Consider the above diagram. In an analytical format, every coordinate of the complex polygon would be included in the file. In a tiled format, the geometry would be split up into every tile it intersects, and possibly simplified as well.
+
+The analytical format is more useful for operations like a spatial join, because the entire geometry is available. It's harder to perform such analyses on tiled data because a program must:
```

Comment:
I find this a little bit confusing. If you have an analytical format that is chunked, you may only read a subset of the data as well and then you don't have the entire geometry available. But maybe I'm misunderstanding this example. Would it be correct to summarize your point that tiled data is not intended for analytical use cases because you would have to reverse engineer the geometry?

Reply:
> If you have an analytical format that is chunked, you may only read a subset of the data as well and then you don't have the entire geometry available

The difference is that in an analytical format, you have a subset of the entire number of _rows_, but for every row, you know that you have the _whole_ geometry, not just a small part of it.